docs/tutorials/node-installation.md

```diff
@@ -60,8 +60,6 @@ tail -f stderr.txt
 
 Check if you can access your node using link `http://**YOUR_SERVER**/v1/chain/get_info`
 
-Here is an API [Example](https://latamlink.eosio.cr/v1/chain/get_info)
```

Comment:
@leisterfrancisco  the correct link here would be : https://lacchain.eosio.cr/v1/chain/get_info

Reply:
Solved